test/blackbox-tests/test-cases/wasmoo/public-libs.t/run.t

```diff
@@ -0,0 +1,3 @@
+Compilation of libraries with public-names
```

Comment:
Is "public-names" the name of a file here? Or just generally speaking of public names?